src/core/congestion_control.c

```diff
@@ -160,59 +160,38 @@ QuicCongestionControlGetSendAllowance(
         //
         SendAllowance = 0;
 
-    } else if (!Connection->Settings.PacingEnabled || !Connection->Paths[0].GotFirstRttSample) {
+    } else if (
+        !TimeSinceLastSendValid ||
+        !Connection->Settings.PacingEnabled ||
+        !Connection->Paths[0].GotFirstRttSample ||
+        Connection->Paths[0].SmoothedRtt < MS_TO_US(15)) {
         //
-        // Pacing is disabled or we don't have an RTT sample yet, so just send
-        // everything we can.
+        // We're not in the necessary state to pace.
         //
         SendAllowance = Cc->CongestionWindow - Cc->BytesInFlight;
 
     } else {
+
         //
-        // Try to pace: if the window and RTT are large enough, the window can
-        // be split into chunks which are spread out over the RTT.
```

Comment:
The "split into chunks which are spread out over the RTT" bit is essential for understanding what our "pacing" is doing and not confusing it with the other type of "pacing", so let's keep it.